src/lxc/conf.c

```diff
@@ -1152,6 +1152,10 @@ static int fill_autodev(const struct lxc_rootfs *rootfs)
 	cmask = umask(S_IXUSR | S_IXGRP | S_IXOTH);
 	for (i = 0; i < sizeof(lxc_devs) / sizeof(lxc_devs[0]); i++) {
 		const struct lxc_devs *d = &lxc_devs[i];
+
+		if (!strcmp(d->name, "console") && !mount_console)
```

Comment:
On Thu, Mar 17, 2016 at 10:31:21AM -0700, Stéphane Graber wrote:
> > @@ -1152,6 +1152,10 @@ static int fill_autodev(const struct lxc_rootfs *rootfs)
> >  	cmask = umask(S_IXUSR | S_IXGRP | S_IXOTH);
> >  	for (i = 0; i < sizeof(lxc_devs) / sizeof(lxc_devs[0]); i++) {
> >  		const struct lxc_devs *d = &lxc_devs[i];
> > +
> > +		if (!strcmp(d->name, "console") && !mount_console)
> 
> hmm, shouldn't that be strcmp(d->name, "console") == 0 && !mount_console ?

there's a ! there, but I can change it to == 0 if you want.